backend/tests/db/test_tests.py

```diff
@@ -32,19 +32,21 @@ def test_basic_filter(
     dbsession: OrmSession,
     worker_id: str | None,
     country: str | None,
-    status: list[StatusEnum] | None,
-    expect: bool,  # noqa
+    statuses: list[StatusEnum] | None,
+    expected: bool,  # noqa: FBT001
```

Comment:
tests all being instanciated via kwargs, I'd have required kwargs instead of the noqa.